docs/high-availability.md

```diff
@@ -8,6 +8,12 @@ Only one controller can run at once. If it crashes, Kubernetes will start anothe
 
 For many users, a short loss of workflow service maybe acceptable - the new controller will just continue running workflows if it restarts.  However, with high service guarantees, new pods may take too long to start running workflows. You should run two replicas, and one of which will be kept on hot-standby.
 
+A workflow pod can disrupt the controller, preventing cron workflows from running and workflows being scheduled. You
+should use:
```

Comment:
```suggestion
A voluntary pod disruption can cause both replicas to be replaced at the same time. You
should use a Pod Disruption Budget to prevent this and Pod Priority to recover faster for an involuntary pod disruption:
```